src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball number is : " + number + "\n"
+               + "Ball color is : " + color + "\n"
+               + "_________________________";
```

Comment:
Using '\n' to create a new line is not the best practice for overriding the toString method. It's better to use String.format or simply concatenate with '+' for better readability and to avoid explicit platform-specific line separators.